kovri.i2p/_i18n/en/acknowledgements.md

```diff
@@ -1,9 +1,9 @@
 In addition to all Boost and OpenSSL contributors, and all coders since time-immemorial who have made this project possible, we would like to acknowledge the following:
 
-- Wei Dai, Jeffrey Walton, and all the developers at Crypto++ for proving an influential, free C++ library of cryptographical library
+- Wei Dai, Jeffrey Walton, and all the developers at Crypto++ for proving an influential, free C++ class library of cryptographic schemes
 - Dean Michael Berris, Glyn Matthews, and all the developers at cpp-netlib for providing an actively developed cross-platform networking library
-- All developers involved in the [ref10 implementation from SUPERCOP](http://bench.cr.yp.to/supercop.html) from which we base our Ed25519 implementation
+- All developers involved in the [ref10 implementation from SUPERCOP](https://bench.cr.yp.to/supercop.html) from which we base our Ed25519 implementation
 - zzz, str4d, and all Java I2P developers both past and present who have set the standard for all other I2P implementations to follow
 - orion for providing ```i2pcpp```: the [original](http://git.repo.i2p.xyz/w/i2pcpp.git) C++ implementation of I2P
-- orignal for providing ```i2pd```: the most feature-complete C++ implementation of I2P for [us to fork from](https://github.com/purplei2p/i2pd/commit/45d27f8ddc43e220a9eea42de41cb67d5627a7d3)
```

Comment:
Orignal is correct and shouldn't be changed, [see here](https://github.com/orignal)